src/c++/uds/userLinux/tests/albtest.c

```diff
@@ -300,6 +301,7 @@ TestResult runTest(const CU_SuiteInfo *suite, const CU_TestInfo *test)
       int status;
       int tmp = waitpid(pid, &status, 0);
       if (tmp == -1) {
+        set_exit_on_assertion_failure(previous);
```

Comment:
I think you are doing more than you need here. There's no need to unset the flag, unit tests expect it to be on all the time anyway.

Reply:
ok i followed your idea of moving the call to main()